edi_oca/models/edi_backend.py

```diff
@@ -445,6 +445,10 @@ def _exchange_process_check(self, exchange_record):
             "input_processed_error",
         ]
 
+    def _update_related_ack(self, exchange_record):
+        if exchange_record.parent_id and exchange_record.type_id.ack_for_type_ids:
```

Comment:
I'm not sure the check on `ack_for_type_ids` is significant. What about this?
```suggestion
        if exchange_record.parent_id.ack_expected:
```